conf/.env

```diff
@@ -0,0 +1,85 @@
+MIX_ENV=prod
+FLAVOUR=classic
+
+WITH_DOCKER=no
+
+## OTHER CONFIG ##
+# server domain name:
+HOSTNAME=__DOMAIN__
+# server port:
+SERVER_PORT=__PORT__
+# port your visitors will access (typically 80 or 443, will be different than SERVER_PORT only if using a reverse proxy)
+PUBLIC_PORT=443
+# what service to use for sending out emails (eg. smtp, mailgun, none) NOTE: you should also set the corresponding keys in secrets section
+MAIL_BACKEND=sendmail
```

Comment:
do you mean `smtp`?

Reply:
I think it can work with Yunohost sendmail backend, and maybe smtp too, but anyway I'll check that later on.